assets/src/scss/layout/_navbar.scss

```diff
@@ -230,23 +304,12 @@ a.nav-link {
   padding: $sp-2 0;
   position: relative;
   width: 100%;
+  text-decoration: none;
 
   @include large-and-up {
     padding: 0;
   }
 
```

Comment:
Removed all this code since the border bottom is assigned to the `:after` pseudo element.

Reply:
Could this be the reason why the underline is now gone in the mobile menu? 🤔

Reply:
The `text-decoration` feature is not used anymore. The underline effect is now applied to the `a`'s `:after` pseudoselector. See [here](https://github.com/greenpeace/planet4-master-theme/blob/fdc54acd1d45f16e90f3f3f7238b5da358787fb2/assets/src/scss/layout/_navbar.scss#L372-L373)

Reply:
Yes I saw that, but there used to be an underline effect in the mobile menu too and that is gone now with this PR, so I was wondering if it could be due to this particular removal

Reply:
Check it again! I rolled back this effect and included to mobile. [Here is the code](https://github.com/greenpeace/planet4-master-theme/pull/1604/files#diff-91009267fa7864d1b148d657a615793b252f3e148c4001b639eab5a4c51324e4R365-R369)